src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -1073,8 +1119,15 @@ int wm_vuldet_send_cve_report(vu_report *report) {
         // Set the alert body
         cJSON_AddItemToObject(alert, "vulnerability", alert_cve);
         cJSON_AddStringToObject(alert_cve, "cve", report->cve);
-        cJSON_AddStringToObject(alert_cve, "title", report->title && *report->title ? report->title : report->rationale);
+        if (report->title) {
```

Comment:
Here there was a very strange assignment for the `title`, that sometimes resulted in a duplicated `rationale` field. Now if we have the `title`, we add it and the check whether the `rationale` is present to be added to the `rationale` field.
If `title` is not present and we have a rationale string, we add the rationale string to the `title`. In this case the field `rationale` is not reported to avoid duplicated information.